hil_extension/yay.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+# yay lets you parse config values in shells scripts from
+# our global ../conf/quads.yml configuration.  This lets us
+# have one authoritative place for configuration across all
+# tooling within QUADS.  Python has a library for this already
+# so this is only needed for shell and expect.
+# Original credit goes to:
+# https://github.com/HumanNeuroscienceLab/gunther/blob/master/include/yay.sh
```

Comment:
Actually, yay.sh is already used by Red-Hat. It is in quads/bin

I made another copy in quads/lib to make my code completely independent of the rest Quads